src/main/java/com/faforever/client/chat/UserInfoWindowController.java

```diff
@@ -333,18 +329,16 @@ private void plotTechBuiltChart(Map<String, PlayerEvent> playerEvents) {
     )));
   }
 
-  @SuppressWarnings("unchecked")
   private void plotGamesPlayedChart() {
-    Player currentPlayer = playerService.getCurrentPlayer().orElseThrow(() -> new IllegalStateException("Player must be set"));
-    leaderboardService.getEntryForPlayer(currentPlayer.getId()).thenAccept(leaderboardEntryBean -> Platform.runLater(() -> {
+    leaderboardService.getEntryForPlayer(player.getId()).thenAccept(leaderboardEntryBean -> Platform.runLater(() -> {
       int ladderGamesCount = leaderboardEntryBean.getGamesPlayed();
-      int custonGamesCount = currentPlayer.getNumberOfGames();
+      int custonGamesCount = player.getNumberOfGames();
       Platform.runLater(() -> gamesPlayedChart.setData(FXCollections.observableArrayList(
           new PieChart.Data(i18n.get("stats.custom"), custonGamesCount),
           new PieChart.Data(i18n.get("stats.ranked1v1"), ladderGamesCount)
       )));
     })).exceptionally(throwable -> {
-      log.warn("Leaderboard entry could not be read for current player: " + currentPlayer.getUsername(), throwable);
+      log.warn("Leaderboard entry could not be read for player: " + player.getUsername(), throwable);
```

Comment:
Better usage would be
`log.warn("Leaderboard entry could not be read for player: {}", player.getUsername(), throwable);`